client/starwhale/api/_impl/model.py

```diff
@@ -23,7 +23,9 @@
 from starwhale.consts.env import SWEnv
 from starwhale.utils.error import NotFoundError
 
-from .loader import DataField, DataLoader, get_data_loader
+from .loader import DataField, DataLoader, get_data_loader, SimpleDataLoader
+from .wrapper import Evaluation, IEvaluation
+from ...core.job.model import Context
```

Comment:
Use absolute import?  We don't recommend the `...` related import.

`from starwhale.core.job.model import Content` ?